### PR TITLE
Jbaker/swap value poc

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #230

## Summary:

Update `examples.js` to support dependency on multiple CEs for a single example before execution.

Add basic functional example with a button swapping the value between two elements.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Docs examples

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**
-- Auro Design System Team

### CODE REVIEW

File: apiExamples/swapValue.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<auro-input id="swapExampleLeft" bordered>
+  <span slot="label">Left Input</span>
+</auro-input>
+<auro-button id="swapExampleBtn">Swap Values</auro-button>
```
Comment (blackfalcon) [inline apiExamples/swapValue.html:4]: When it comes to the button, I don't see why we simply can't do something like the following:

```html
<button>
   <svg viewBox="0 0 24 24" width="24px" height="24px" 

           aria-hidden="true" focusable="false">
        <!-- svg content -->
    </svg>
    <span class="visually-hidden">Swap input values</span>

</button>
```

Icons are already set to be a specific height that works with all designs and we just need to ensure that we are accessible. There is no dependency on any other kind of component update that I can see necessary.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: demo/demo.md [hunk 2]
```diff
@@ -766,3 +766,38 @@ For use cases where the field is `required`, but live validation is not wanted,
...
+  <span slot="label">Left Input</span>
+</auro-input>
+<auro-button id="swapExampleBtn">Swap Values</auro-button>
+<auro-input id="swapExampleRight" bordered>
+  <span slot="label">Right Input</span>
+</auro-input>
+```
+<!-- The below code snippet is automatically added from ./../apiExamples/swapValue.html -->
+<!-- AURO-GENERATED-CONTENT:END -->
+  ```
```
[the remark this answers is not in the document]
Comment (jason-capsule42) [reply demo/demo.md]: Removed the extra code ticks.

File: demo/demo.md [hunk 3]
```diff
@@ -766,3 +766,54 @@ For use cases where the field is `required`, but live validation is not wanted,
   </auro-input>
   ```
 </auro-accordion>
+
+## Swapping Values Between Inputs
+
+<div class="exampleWrapper exampleWrapper--flex">
```
Comment (blackfalcon) [inline demo/demo.md:772]:
```suggestion
## Swapping Values Between Inputs

The following example illustrates how you can easily use the auro-input element combined with some additional JavaScript to easily create an experience where a user can easily swap the value from one input element to the other. 

<div class="exampleWrapper exampleWrapper--flex">
```

File: apiExamples/swapValue.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<auro-input id="swapExampleLeft" bordered>
```
Comment (blackfalcon) [inline apiExamples/swapValue.html:1]:
```suggestion
<auro-input id="swapExampleLeft" bordered value="Use the button to swap this value">
```

File: demo/demo.md [hunk 5]
```diff
@@ -766,3 +766,54 @@ For use cases where the field is `required`, but live validation is not wanted,
...
+
+  btn.addEventListener('click', () => {
+    const valueOne = inputOne.value;
+    const valueTwo = inputTwo.value;
+
+    inputOne.value = valueTwo;
+    inputTwo.value = valueOne;
+  });
+}
+```
```
Comment (blackfalcon) [inline demo/demo.md:817]: There should be a comment in here that states that this optional javascript can be placed anywhere in their app.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: apiExamples/swapValue.html (hunk 1 above)
Comment (jason-capsule42) [reply apiExamples/swapValue.html:4]: Resolving without direct change due to review comments being added to a draft PR that was not ready for review.

File: apiExamples/swapValue.html (hunk 4 above)
Comment (jason-capsule42) [reply apiExamples/swapValue.html:1]: Resolving without direct change due to review comments being added to a draft PR that was not ready for review.

File: demo/demo.md (hunk 5 above)
Comment (jason-capsule42) [reply demo/demo.md:817]: Resolving without direct change due to review comments being added to a draft PR that was not ready for review.

File: demo/demo.md (hunk 3 above)
Comment (jason-capsule42) [reply demo/demo.md:772]: Resolving without direct change due to review comments being added to a draft PR that was not ready for review.

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**